saleor/graphql/product/tests/test_collection.py

```diff
@@ -920,6 +1018,40 @@ def test_remove_products_from_collection(
     assert data["products"]["totalCount"] == no_products_before - len(product_ids)
 
 
+@patch("saleor.plugins.manager.PluginsManager.product_updated")
+def test_remove_products_from_collection_trigger_product_updated_webhook(
+    product_updated_mock,
+    staff_api_client,
+    collection,
+    product_list,
+    permission_manage_products,
+):
+    query = """
+        mutation collectionRemoveProducts(
+            $id: ID!, $products: [ID]!) {
+            collectionRemoveProducts(collectionId: $id, products: $products) {
+                collection {
+                    products {
+                        totalCount
+                    }
+                }
+            }
+        }
+    """
+    collection.products.add(*product_list)
+    collection_id = to_global_id("Collection", collection.id)
+    product_ids = [to_global_id("Product", product.pk) for product in product_list]
+    no_products_before = collection.products.count()
```

Comment:
The name suggests that the value here is 0.  Not sure that it is correct as, in line 1051, we have subtraction from this value.